extensions/scarb-doc/src/docs_generation.rs

```diff
@@ -153,3 +153,35 @@ impl_doc_item!(TraitType);
 impl_doc_item!(TraitFunction);
 impl_doc_item!(TypeAlias);
 impl_doc_item!(Variant);
+
+trait ItemsName<'a> {
+    const ITEMS_NAME: &'a str;
+}
+
+macro_rules! impl_items_name {
+    ($t:ty, $name:expr) => {
+        impl<'a> ItemsName<'a> for $t {
+            const ITEMS_NAME: &'a str = $name;
+        }
+    };
+}
```

Comment:
1. wdyt?

```suggestion
trait ItemsName {
    const HEADER: &'static str;
}

macro_rules! impl_items_name {
    ($t:ty, $name:expr) => {
        impl ItemsName for $t {
            const HEADER: &'_ str = $name;
        }
    };
}
```

2. Why can't this be part of `DocItem` trait?